<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DGN-AM测试</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>设置</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>端口</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtPort">
         <property name="text">
          <string>2000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSet">
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <pointsize>23</pointsize>
       </font>
      </property>
      <property name="title">
       <string>通道1</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <pointsize>23</pointsize>
            </font>
           </property>
           <property name="text">
            <string>频率</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_frq_1">
           <property name="font">
            <font>
             <pointsize>23</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0000000</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <pointsize>23</pointsize>
            </font>
           </property>
           <property name="text">
            <string>功率</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_pow_1">
           <property name="font">
            <font>
             <pointsize>23</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0000000</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <pointsize>23</pointsize>
            </font>
           </property>
           <property name="text">
            <string>调幅度</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_tfu_1">
           <property name="font">
            <font>
             <pointsize>23</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btnPlay1">
         <property name="font">
          <font>
           <pointsize>23</pointsize>
          </font>
         </property>
         <property name="text">
          <string>播放</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btnStop1">
         <property name="font">
          <font>
           <pointsize>23</pointsize>
          </font>
         </property>
         <property name="text">
          <string>停止</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="font">
       <font>
        <pointsize>23</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>通道2</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <pointsize>23</pointsize>
            </font>
           </property>
           <property name="text">
            <string>频率</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_frq_2">
           <property name="font">
            <font>
             <pointsize>23</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0000000</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_8">
           <property name="font">
            <font>
             <pointsize>23</pointsize>
            </font>
           </property>
           <property name="text">
            <string>功率</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_pow_2">
           <property name="font">
            <font>
             <pointsize>23</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0000000</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_9">
           <property name="font">
            <font>
             <pointsize>23</pointsize>
            </font>
           </property>
           <property name="text">
            <string>调幅度</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_tfu_2">
           <property name="font">
            <font>
             <pointsize>23</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0000000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btnPlay1_2">
         <property name="font">
          <font>
           <pointsize>23</pointsize>
          </font>
         </property>
         <property name="text">
          <string>播放</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btnStop1_2">
         <property name="font">
          <font>
           <pointsize>23</pointsize>
          </font>
         </property>
         <property name="text">
          <string>停止</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
